.github/workflows/cypress-testing.yml

```
@@ -118,6 +150,133 @@
+          # Start Hatchet Engine in background
+          echo "🔄 Starting Hatchet Engine..."
+          docker run -d --name hatchet_engine --network host \
+            -v hatchet_config:/hatchet/config \
+            -v hatchet_certs:/hatchet/certs \
+            -e DATABASE_URL='postgres://hatchet:hatchet@localhost:5435/hatchet' \
+            -e SERVER_GRPC_BIND_ADDRESS='0.0.0.0' \
+            -e SERVER_GRPC_INSECURE='t' \
+            ghcr.io/hatchet-dev/hatchet/hatchet-engine:latest \
+            /hatchet/hatchet-engine --config /hatchet/config
+
+          # Start Hatchet Dashboard in background  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)